l2tdevtools/source_helper.py

```diff
@@ -359,3 +358,14 @@ def GetProjectVersion(self):
           self.project_name, version_definition)
 
     return self._project_version
+
+  def GetSourcePackageFilename(self):
+    """Retrieves the filename of the source package.
```

Comment:
Please add a note that this will also attempt to download the package, not just fetching the filename.

Reply:
done